pkg/api/serialization_test.go

```diff
@@ -375,7 +375,9 @@ func roundTrip(t *testing.T, codec runtime.Codec, originalItem runtime.Object) {
 var skipStandardVersions = map[string][]string{
 	// The API versions here are to test our object that serializes from/into
 	// docker's registry API.
-	"DockerImage": {"pre012", "1.0"},
+	"DockerImage":          {"pre012", "1.0"},
+	"DeploymentConfigList": {"", "v1"},
```

Comment:
This is because we're breaking compatibility with `v1beta3`?  Where are the special cased tests to make sure I can still correctly parse these from `v1beta3` in?